test/change.test.js

```diff
@@ -601,10 +610,90 @@ describe('Change', function() {
           modelName: updateRecord.modelName,
           prev: null, // this is the current local revision
           rev: 'new-rev',
+          tenant: null,
         });
 
         done();
       });
     });
   });
 });
+
+describe('Change with multi tenancy', function() {
+  var ChangeWithTenant;
+  var TestModelWithTenant;
+  var tenantProperty = 'tenantId';
+
+  beforeEach(function() {
+    var memory = loopback.createDataSource({
+      connector: loopback.Memory
+    });
+    TestModelWithTenant = loopback.PersistedModel.extend('ChangeTestModel',
+      {
+        id: {id: true, type: 'string', defaultFn: 'guid'}
+      },
+      {
+        trackChanges: true,
+        tenantProperty: tenantProperty
```

Comment:
This is more generic than needed in the test. We are not testing a multi-tenancy mixin, just verifying that it's possible to propagate properties from a model to the change model.

For the purpose of this test, please assume a hard-coded `tenant` property name, e.g. `tenantId`.